CONTRIBUTING.md

```diff
@@ -70,6 +72,15 @@ open http://localhost:8080
 
 `npm run dev` runs a [webpack-dev-server](https://webpack.github.io/docs/webpack-dev-server.html) instance that uses `demos/` as its content base. This should aid you in initial development of a component. It's served on port 8080.
 
+#### Appengine developement server
+
+```
+MDC_ENV=development npm run build && gcloud app deploy app.yaml --project google.com:mdc-web-dev --version $USER
+gcloud app browse
+```
+
+The above script will build and deploy the app to mdc-web's dev server with your userid as its version number, you can switch to your version by adding $USER-dot to the URL. This would be helpful if we need to share work-in-progress work within teams and designers.
```

Comment:
The URL that we had an agreement that we shouldn't put it in the doc.

Shall I rephrase this to be
> you can switch to your version by prepend `$USER-dot-` to the URL opened when you run `gcloud app browse`

Does this make it a bit clearer?